test/test_cost.py

```diff
@@ -2724,18 +3368,36 @@ def test_redq(self, delay_qvalue, num_qvalue, device, td_est):
                     )
                 )
             elif k == "loss_qvalue":
-                assert all(
+                assert not any(
```

Comment:
has the default behaviour of REDQ changed?

Reply:
I restored the defaults, I changed it to False